src/test_hash_table.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+
+
+def test_hash():
+    """Test hash function."""
+    from hash_table import HashTable
+    new_hash = HashTable(100)
+    hashed = new_hash._hash('string')
+    assert hashed < len(new_hash.table)
+    assert isinstance(hashed, int)
+
```

Comment:
Need to test the length of the hash table is the length that you initialized with.